app/uk/gov/hmrc/eusubsidycompliancefrontend/views/AddPublicAuthorityPage.scala.html

```diff
@@ -55,6 +56,14 @@
 
     @formHelper(action = controllers.routes.SubsidyController.postAddClaimPublicAuthority) {
 
+      @H1(messages("add-claim-public-authority.title"))
```

Comment:
you can add a check for the title as well

Reply:
good spot, I have addressed this in latest commit!